commit bb96486b127054e7b0e02c467b0f61a20d227161
Author: Juan RP <xtraeme@gmail.com>
Date:   Thu Jun 27 08:02:19 2019 +0200

    xbps-install: make sure to call xbps_end() before exiting.

diff --git a/bin/xbps-install/main.c b/bin/xbps-install/main.c
index 371f26c8..3622af5a 100644
--- bin/xbps-install/main.c
+++ bin/xbps-install/main.c
@@ -263,7 +263,7 @@ main(int argc, char **argv)
 			}
 		}
 		if (eexist == argc)
-			return 0;
+			goto out;
 
 		rv = exec_transaction(&xh, maxcols, yes, drun);
 	} else if (!update) {
@@ -280,11 +280,12 @@ main(int argc, char **argv)
 			}
 		}
 		if (eexist == argc)
-			return 0;
+			goto out;
 
 		rv = exec_transaction(&xh, maxcols, yes, drun);
 	}
 
+out:
 	xbps_end(&xh);
 	exit(rv);
 }
